tools/profileBuilder/cmd/root.go

```diff
@@ -62,6 +63,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&modulesFlag, "modules", "m", false, "Executes commands in modules-aware mode.")
 	rootCmd.PersistentFlags().StringVarP(&profileName, "name", "n", "", "The name that should be used to identify the profile.")
 	rootCmd.PersistentFlags().StringVarP(&outputRootDir, "output-location", "o", "", "The folder in which to output the generated profile.")
+	rootCmd.PersistentFlags().IntVar(&semLimit, "sem-limit", 8, "The limit of the maximum goroutines")
```

Comment:
Can we not make this a flag?  More flags make the tool confusing plus is anybody really going to change this?

Reply:
Ok we can leave this, but perhaps a better description.  "The maximum number of packages to concurrently build." or something

Reply:
Done